h/zos.h

```diff
@@ -1123,6 +1123,44 @@ typedef struct TIOTEntry_tag{
   int       tioejfcb;      /* this evil thing is 24 high bits of SWA index, and last byte of status bits */
 } TIOTEntry;
 
+/* IDTA Version */
+#define IDTA_VERSION_0001       0x0001  /* Version 1 of IDTA */
+#define IDTA_CURRENT_VERSION    0x0001
+
+#define IDTA_IDT_BUFFER_LEN_MIN 1024    /* Token Buffer Length Minimum Size (1024) */
+
+/* IDTA idt_type */
+#define IDTA_JWT_IDT_Type       0x0001  /* Indicates the token is a JWT */
+
+/* IDTA idt_prop_out flag */
+#define IDTA_SAF_IDT_RETURN     0x8000  /* Token was returned by SAF */
+#define IDTA_IDT_AUTH_DONE      0x4000  /* Token Authentication Complete */
+#define IDTA_IDT_SIGNED         0x2000  /* Token has a signature */
+
+/* IDTA idt_prop_in flag */
+#define IDTA_End_User_IDT       0x8000  /* Token is input from an end user */
+
+/* Idenitity Token Generation Return Code */
+#define IDTA_IDT_GEN_RC_SUCC          0
+#define IDTA_IDT_GEN_RC_UNSIGN        3
+#define IDTA_IDT_GEN_RC_ICSF_UNAVAIL  4
+#define IDTA_IDT_GEN_RC_ICSF_ERR      5
+
+
+typedef struct Idta_tag{
```

Comment:
Please follow the style of the source you're modifying, specifically:
* `Idta` should be `IDTA`
* No snake case in field names

Reply:
Fixed